config/config.go

```diff
@@ -104,6 +104,8 @@ type Config struct {
 	//Prometheus
 	PrometheusPushGateway *PrometheusPushGatewayConfig
 
+	CoinPolicyMigrationConfig *CoinPolicyMigrationConfig
```

Comment:
Rename this field to `CoinPolicyMigration`

Reply:
Done.